db/migrate/20181018152839_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :url
+      t.references :user, foreign_key: true
```

Comment:
There are cases where you don't know from the beginning you need a relation to another model. If that's the case [you can use a migration (and a generator) just to add the reference.](https://edgeguides.rubyonrails.org/active_record_migrations.html#creating-a-migration)

E.g. `rails generate migration AddUserRefToProducts user:references`

Reply:
Thanks. I am usually stumped with those kind of things.

Reply:
Everything looks good. Just add a test to check that the post has a user_id (similar to what you did in the test for comments and merge)